cmd/main.go

```diff
@@ -69,7 +70,8 @@ var (
 	// ProxmoxTokenID env variable that defines the Proxmox token id.
 	ProxmoxTokenID string
 	// ProxmoxSecret env variable that defines the Proxmox secret for the given token id.
-	ProxmoxSecret string
+	ProxmoxSecret             string
+	ProxmoxReserveOnlyRunning string
```

Comment:
The name of this variable isn't particularly descriptive and it's missing a comment.
The type should be `bool`.